api-documentation/payments.md

```diff
@@ -99,7 +103,7 @@ Payee email address.
 {% endapi-method-parameter %}
 
 {% api-method-parameter name="phone\_number" type="string" required=false %}
-Payee phone number.
+Payee phone number. It is expected to provide the full phone number including international prefix (e.g. +31)
```

Comment:
```suggestion
Payee phone number. It is expected that partner provides the phone number in E.164 format, i.e. including international prefix (e.g. +31)
```

Reply:
I think it's better to be explicit about the E.164 format, to not leave space for misunderstandings about what a "full phone number" is.